src/main/java/com/example/bankservicedemo/service/AccountService.java

```diff
@@ -0,0 +1,14 @@
+package com.example.bankservicedemo.service;
+
+import com.example.bankservicedemo.model.Account;
+import java.util.List;
+
+public interface AccountService {
+    Account save(Account account);
+
+    List<Account> getAccountsByPhone(String phone);
+
+    Account getByAccountNumber(String accountNumber);
+
+    void blockAccount(String accountNumber);
```

Comment:
let's set `active` to false somewhere in controller and then just call `service.save()` - what do you think?